src/wallet/sqlite.cpp

```diff
@@ -233,6 +233,15 @@ void SQLiteDatabase::Open()
         throw std::runtime_error(strprintf("SQLiteDatabase: Failed to enable fullfsync: %s\n", sqlite3_errstr(ret)));
     }
 
+    if (gArgs.GetBoolArg("-unsafesqlitesync", false)) {
+        // Use normal synchronous mode for the journal
+        LogPrintf("SQLiteDatabase: WARNING SQLite is configured to not wait for data to be flushed to disk. Data loss and corruption may occur.");
```

Comment:
```suggestion
        LogPrintf("WARNING SQLite is configured to not wait for data to be flushed to disk. Data loss and corruption may occur.\n");
```

function names and file paths are already logged in tests. Also missing `\n` to make linter happy?

Reply:
Done